src/world.ts

```diff
@@ -189,36 +242,37 @@ export class World {
     this.app.ticker.add((deltaTime) => {
       this.update(deltaTime);
     });
+
+    // Ensure our world is correctly sized
     this.app.resize();
-    // Container that will hold our room "furniture" elements,
-    // like broadcast spots
-    this.furnitureContainer = new PIXI.Container();
```

Comment:
Moved this part into the constructor of the World class, to sit alongside the user container for consistency.